src/Story.php

```diff
@@ -29,47 +32,134 @@ final public static function load(): self
     }
 
     /**
+     * Get a random item from a pool.
+     */
+    final public static function getRandom(string $pool): Proxy
+    {
+        return static::getRandomSet($pool, 1)[0];
+    }
+
+    /**
+     * Get a random set of items from a pool.
+     *
+     * @return Proxy[]
+     */
+    final public static function getRandomSet(string $pool, int $number): array
+    {
+        if ($number < 0) {
```

Comment:
Just wondering: is there any reason to allow "0" as $number?

Reply:
I changed to `$number < 1`.